File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;

    do
    {
        n = get_int("Height: "); //get height
    }
    while (n < 1 || n > 8); //if number is not within the 1-8 range, repeat prompt

    for (int i = 1; i <= n; i++) //creates pyramid height
    {
        for (int b = 0; b < (n - i); b++) //creates spaces
        {
            printf(" ");
        }

        for (int c = 0; c < i; c++) //creates hashes
        {

            printf("#");
        }

        printf("  ");

        for (int c = 0; c < i; c++) //creates adjacent hashes
        {
            printf("#");
        }

        printf("\n"); //move prompt to next line
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:  # get height
    height = get_int("Height: ")

    if 1 <= height <= 8:  # meet height requirements
        break

for i in range(1, height + 1, 1):  # create rows in pyramid
    for j in range(height - i):  # draw spaces
        print(" ", end="")

    for j in range(i):  # draw L hashes
        print("#", end="")

    print("  ", end="")

    for j in range(i):  # draw R hashes
        print("#", end="")

    print()